packages/taquito-contracts-library/src/taquito-contracts-library.ts

```diff
@@ -5,7 +5,7 @@
 
 import { EntrypointsResponse, ScriptedContracts } from '@taquito/rpc';
 import { Extension, Context } from '@taquito/taquito';
-import { validateAddress, ValidationResult, invalidErrorDetail } from '@taquito/utils';
+import { validateAddress, ValidationResult, invalidDetail } from '@taquito/utils';
```

Comment:
just a nitpick, but I noticed that instead of errorDetail we're using `invalidDetail` for validation. Maybe it's okay to use `errorDetail` for this as well? Slightly easier to keep track of 1 variable that's consistent throughout the codebase for error message details. What do you think?